src/app/clusters/energy-evse-server/energy-evse-server.h

```diff
@@ -81,6 +100,41 @@ class Delegate
      */
     virtual Protocols::InteractionModel::Status StartDiagnostics() = 0;
 
+    /**
+     * @brief Delegate should implement a handler to SetTargets command.
+     * It should report Status::Success if successful and may
+     * return other Status codes if it fails
+     */
+    virtual Protocols::InteractionModel::Status
+    SetTargets(const DataModel::DecodableList<Structs::ChargingTargetScheduleStruct::DecodableType> &) = 0;
+
+    /**
+     * @brief Delegate should implement a handler to PrepareGetTargets
+     *
+     * This needs to load any stored targets into memory and hold it until the GetTargetsFinished is
+     * called by the cluster server.
+     *
+     * @param  Reference to EvseTargetIterator class that implements the ability
```

Comment:
Is this an out param?  Should be documented.

Reply:
Done